DiceKit/FrequencyDistribution.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  FrequencyDistribution.swift
+//  DiceKit
+//
+//  Created by Brentley Jones on 7/24/15.
+//  Copyright © 2015 Brentley Jones. All rights reserved.
+//
+
+import Foundation
+
+public struct FrequencyDistribution: Equatable {
+    
+    public typealias Value = Int
+    public typealias Outcome = Double
+    public typealias OutcomesPerValue = [Value: Outcome]
```

Comment:
Yeah, outcome is the other name for what we are calling "Value". I think I can rename it.